cypress/integration/seededFlows/TopNavigationFlow/MemberDropDownButton.spec.js

```diff
@@ -0,0 +1,81 @@
+describe('Member Menu Dropdown', () => {
```

Comment:
nitpick(blocking): it looks like the other flows in this directory start with a lower case letter and are plural, "topNavigationFlows" seems like it would be least surprising.